.github/workflows/docker-test-and-push.yaml

```
@@ -43,7 +43,7 @@ jobs:
       - name: "run tests on qgep_build (structure)"
         run: |
           docker start qgep
-          docker exec -e PGSERVICE=qgep_build qgep pytest --ignore test/test_import.py
+          docker exec -e PGSERVICE=qgep_build qgep pytest --ignore test/test_import.py --ignore test/test_swmm.py

```

Why do we ignore this test?

---

the new test runs in the `run tests on qgep_build_pum (production)` step just after that one, as it relies on the demo data

---

```suggestion
          # Run tests that depend on data model. Tests that require demo data are ignored and run below in qgep_build_pum
          docker exec -e PGSERVICE=qgep_build qgep pytest --ignore test/test_import.py --ignore test/test_swmm.py
```

Can we add that for clarity?

---

done, added the same to the travis file too